portality/tasks/helpers/background_helper.py

```diff
@@ -64,11 +64,13 @@ def execute_by_job_id(job_id, task_factory: TaskFactory):
     BackgroundApi.execute(task)
 
 
-def execute_by_bg_task_type(bg_task_type: Type[BackgroundTask], **prepare_kwargs):
+def execute_by_bg_task_type(bg_task_type: Type[BackgroundTask], job_wrapper=None, **prepare_kwargs):
     """ wrapper for execute by BackgroundTask
     """
     user = app.config.get("SYSTEM_USERNAME")
     job = bg_task_type.prepare(user, **prepare_kwargs)
+    if job_wrapper is not None:
+        job = job_wrapper(job)
```

Comment:
Yes, good point.  I used it in an earlier iteration of the code, and then factored the upstream code out.  My use case was that when running from the command line, I want to wrap the job with a class which outputs audit messages to stdout.

My inclination, therefore, is to leave this in even though it isn't currently used, as it may be useful.  But could also just clean it up for simplicity.  Thoughts?